doc/dev/index.rst

```diff
@@ -1274,6 +1274,23 @@ there::
 This will keep the teuthology machine, the logs and the packages-repository
 instance but nuke everything else.
 
+Upload logs to archive server
+-----------------------------
+
+Since the teuthology instance in OpenStack is only semi-permanent, with limited
+space for storing logs, ``teuthology-openstack`` provides an ``--upload``
+option which, if included in the ``ceph-workbench ceph-qa-suite`` command,
+will cause logs from all failed jobs to be uploaded to the log archive server
+maintained by the Ceph project. The logs will appear at the URL::
+
+    http://qa-proxy.ceph.com/teuthology/$RUN
+
+where ``$RUN`` is the name of the run - which will be something like
+"ubuntu-2016-07-23_16:08:12-rados-hammer-backports---basic-openstack"
+
+If you don't providing the ``--upload`` option, all the logs can still be found
+on the teuthology VM in the directory ... (?)
```

Comment:
haha - it tells me:

```
# logs in /usr/share/nginx/html
```

But the disk space on the teuthology machine is limited, eh?